modules/vaos/app/services/vaos/v1/fhir_service.rb

```diff
@@ -9,33 +9,47 @@ module V1
     # FHIR (DSTU 2) based REST service http://hl7.org/fhir/dstu2/http.html
     #
     # @example Create a service and fetch an Organization resource by id
-    #   service = VAOS::V1::FHIRService.new(user)
-    #   response = service.read(:Organization, 987654)
+    #   service = VAOS::V1::FHIRService.new(user, :Organization)
+    #   response = service.read(987654)
     #
     class FHIRService < VAOS::BaseService
       STATSD_KEY_PREFIX = 'api.vaos.fhir'
       RESOURCES = %i[Appointment HealthcareService Location Organization Patient Schedule Slot].freeze
 
+      def initialize(user, resource_type)
```

Comment:
NITPICK: if we make user, the second argument, defaulted to nil, then some of these actions that don't require user could later be refactored to not have it. Alternately, consider making it a "named parameter,"

```ruby
def initialize(resource_type, user: nil)
```